### ISSUE TITLE
Metric names in specification don't match hostmetrics's paging scraper

### ISSUE THREAD

**Issue by crobert-1** [open-telemetry/opentelemetry-specification#2727]
Metric names should be consistent between specification and receiver scrapers. The metrics in question are from the `hostmetrics` receiver, specifically the paging scraper: `system.paging.operations.page_in` and `system.paging.operations.page_out`. The specification incorrectly labels these as `system.paging.operations.in` and `system.paging.operations.out`. I believe the README and code are correct and the specification needs to be updated to match, but I'm open to correction here.

The mismatch appeared as a part of the [direction attribute removal](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11815) for the [paging scraper](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11818).

[Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systempaging---pagingswap-metrics)
[hostmetrics receiver paging scraper README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/pagingscraper/documentation.md)
[Relevant code showing page_in and page_out](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/pagingscraper/internal/metadata/generated_metrics_v2.go)

**Comment by rbailey7210** [comment]
Thank you for reporting this. Would you be willing to follow-up?

**Comment by crobert-1** [comment]
I'm unsure how to proceed based on #2726. It looks like discussion is going on to may revert these changes and revisit the metric format change. Reverting the changed would resolve this issue.

**Comment by jmacd** [comment]
Yes, it looks like we will revert all changes stemming from the change of "direction", which I think means reverting the name change here.

**Comment by ocelotl** [comment]
I asked Claude to review this issue and Claude suggests it be closed with this explanation:

> The reported mismatch (spec's .in/.out vs scraper's .page_in/.page_out) stemmed from the 'direction' attribute removal. Maintainers explicitly said they would revert all direction-removal changes (jmacd, crobert-1 comments). The current semantic convention (now in open-telemetry/semantic-conventions, docs/system/system-metrics.md) uses a single metric `system.paging.operations` with a `system.paging.direction` (in/out) attribute, which is the reverted state that removes the mismatch. Also moved-repo: system metrics now live in semantic-conventions.

To guard against a mistaken suggestion, Claude then independently re-verified this conclusion:

> Maintainers said they'd revert the direction-removal (#2727 comments jmacd/crobert-1, re #2726); that revert stuck: semconv system-metrics.md now uses single metric system.paging.operations + system.paging.direction attr, and collector-contrib pagingscraper likewise emits system.paging.operations w/ direction attr, so the .in/.out vs .page_in/.page_out metric-name mismatch is gone. Content moved to semantic-conventions. Issue still OPEN (no closing PR in timeline) but ask is resolved.

Claude evaluated this issue with high confidence that it can be closed.

Claude has classified this issue as resolved.

The intention of this comment is to make it easier for a human being to review this issue before making a decision.

**Comment by crobert-1** [comment]
This can be closed, there are no longer any `system.paging.operations.page_in` and `system.paging.operations.page_out` metrics, I don't believe anything needs updated anymore.
